-new-material-templates/release-notes-template/antora.yml

```diff
@@ -1,83 +1,14 @@
 name: tinymce
```

Comment:
@FarzadHayat : What was the reason to remove the existing `antora.yml` example content?

Reply:
The intention was to shorten the template file to just the attributes that may need to be updated with a new major/minor release. Would you rather keep the full template?

Reply:
File has been removed as it was decided that it is no longer useful. When creating a new major release, the entire branch along with the actual `antora.yml` file will be cloned.